src/test/groovy/org/araneforseti/boundary/fields/BooleanFieldTest.groovy

```diff
@@ -1,14 +1,17 @@
 package org.araneforseti.boundary.fields
 
+import org.araneforseti.boundary.util.MessageConfiguration
 import org.junit.Test
 
 import static org.araneforseti.boundary.TestUtil.scenario_messages_contains
 import static org.araneforseti.boundary.TestUtil.scenarios_contains_value
 
 class BooleanFieldTest {
     String messageName = "otherName"
-    BooleanField requiredBoolean = new BooleanField("testField", true, messageName)
-    BooleanField optionalBoolean = new BooleanField("testField", false, messageName)
+    String fieldType = "boolean"
+    String fieldName = "testField"
```

Comment:
These two fields don't interact with the code under test, yet their values are expected. It should be more explicit what the relationship is